hrjobcontract/js/src/job-contract/controllers/modal/modal-contract.js

```diff
@@ -554,6 +556,22 @@ define([
           $modalInstance.close();
         }
       }
+
+      /**
+       * fetchHealthPlanTypes Fetches and assigns the updated Health Plan Types
+       */
+      function fetchHealthPlanTypes() {
+        ContractHealthService.getOptions(null, true)
+        .then(function (results) {
```

Comment:
Give meaningful names to argument. You just fetched, i assume, the health plans? Then call the argument `healthPlans`, or something like that